src/Cdn/Cdn/help/New-AzCdnDeliveryPolicy.md

```diff
@@ -0,0 +1,92 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Cdn.dll-Help.xml
+Module Name: Az.Cdn
+online version:
+schema: 2.0.0
+---
+
+# New-AzCdnDeliveryPolicy
+
+## SYNOPSIS
+Creates a delivery policy.
+
+## SYNTAX
+
+```
+New-AzCdnDeliveryPolicy [-Description <String>] -Rule <PSDeliveryRule[]>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **New-AzCdnDeliveryPolicy** cmdlet creates a delivery policy for CDN endpoint creation.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-AzCdnDeliveryPolicy -Description "Sample Policy" -Rule $rule
```

Comment:
@hytao for the examples you're adding in the new cmdlets, please make sure that the corresponding output that the user would see when running the cmdlet is displayed in the code block as well